tools/Modules/Run-UnitTests.ps1

```diff
@@ -0,0 +1,8 @@
+#Requires -Modules Pester
+$defaults = [System.IO.Path]::GetDirectoryName($PSCommandPath)
+Start-Job -ScriptBlock { 
+    Import-Module $using:PSCommandPath\..\..\..\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Profile\AzureRM.Profile.psd1;
```

Comment:
Don't you want to use PSScriptRoot here instead of PSCommandPath?

Reply:
done